docs/user/alerting/stack-alerts/es-query.asciidoc

```diff
@@ -6,20 +6,20 @@ The ES query alert type is designed to run a user-configured {es} query over ind
 actions to run when the threshold condition is met.
 
 [float]
-==== Creating the alert
+==== Create the alert
```

Comment:
The ES query alert type runs a user-configured Elasticsearch query, compares the number of matches to a configured threshold, and schedules actions to run when the threshold condition is met.

I removed "over indices".  Is it needed?

Reply:
For consistency, how about this for the headings:

Create the alert
Define the conditions
Add action variables
Test your query